Header header

#Payloads to add to process controller
Payload[] payloads
#Payloads targets to add to process controller
PayloadTarget[] payload_targets
#Link markers to add to process controller
LinkMarkers[] link_markers
#Payload, PayloadTarget, and LinkMarkers ids to remove from controller
string[] delete_payloads
